src/unstract/sdk/adapters/embedding/no_op_embedding/src/static/json_schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "NoOp Embedding Adapter",
+  "type": "object",
+  "required": [
+    "adapter_name",
+    "wait_time"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "Name",
+      "default": "",
+      "description": "Provide a unique name for this No Op adapter instance. Example: no-op-instance-1"
+    },
+    "wait_time": {
```

Comment:
@harini-venkataraman I hope this works as expected for floats as well

Reply:
@chandrasekharan-zipstack I doubt if we need to support float for wait time. 
@hari-kuriakose can you please help here?

Reply:
@harini-venkataraman if its a simple change / just works - it'll be a good to have. Do we parse to int anywhere?

Reply:
@harini-venkataraman No need to ensure support of float for now.

Reply:
@chandrasekharan-zipstack We dont explicitly parse, but have static typing.
I just noticed, sleep itself takes an argument in float. Changing the typing to float.

Reply:
@chandrasekharan-zipstack We dont explicitly parse, but have static typing.
I just noticed, sleep itself takes an argument in float. Changing the typing to float.

Reply:
@hari-kuriakose Mere changing typing works good for float as well. JSON Schema already has `number` datatype.